src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -584,10 +562,13 @@ class MediaBaseViewer extends BaseViewer {
             this.setMediaTime(start);
         }
         if (arguments.length === 0 || hasValidStart) {
-            this.mediaEl.play();
+            const autoPlayPromise = this.mediaEl.play();
```

Comment:
does `handleRate` and `handleVolume` need to wait for play to resolve if a promise?

Reply:
No, because rate and volume are properties of `mediaEl` which can be set before or after playback.